docs/src/openapi-v1.yaml

```diff
@@ -20,6 +20,98 @@ tags:
 - name: transaction
 - name: paras
 paths:
+  /accounts/{accountId}/asset-balances:
+    get:
+      tags:
+      - accounts
+      summary: Get an array of asset-balances for an account.
+      description: Returns information about an accounts asset-balances. This is 
+        specific to the assets-pallet for parachains. If no `assets` query parameter
+        is provided, all asset-balances for the given account will be returned. 
+      operationId: getAssetBalances
+      parameters:
+      - name: accountId
+        in: path
+        description: SS58 address of the account.
+        required: true
+        schema:
+          type: string
+          pattern: '^[1-9A-HJ-NP-Za-km-z]{8,64}$'
+      - name: at
+        in: query
+        description: Block at which to query asset-balance info for the 
+          specified account.
+        required: false
+        schema:
+          type: string
+          description: Block height (as a non-negative integer) or hash
+            (as a hex string).
+          format: unsignedInteger or $hex
+      - name: assets
+        in: query
+        description: An array of AssetId's to be queried. If not inputed by default
+          all asset-balances associated with the `accountId` will be returned. The array
+          format follows Express 4.x API recommendations. ex:`?assets[]=1&assets[]=2&assets[]=3`.
+        required: false
+        schema:
+          type: string
+          description: An array of assetId numbers represented as strings
+          format: Array of unsignedInteger's
+      responses:
+        "200": 
+          description: successfull operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountAssetsBalances'
+  /accounts/{accountId}/asset-approvals:
+    get:
+      tags:
+      - accounts
+      summary: Get an asset-approval for an account.
+      description: Returns information about an account's asset-approval transaction.
+        It is required to pass in a delegate and an assetId as query parameters.
```

Comment:
Are these line breaks working when displaying the UI? I forget exactly but sometimes these line breaks don't play way with openapi yaml.
If not you can do:
```
description: |
    80 chars of text
    more text